main/tests/server/src/com/google/refine/expr/functions/CoalesceTests.java

```diff
@@ -33,38 +33,16 @@ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 
 package com.google.refine.expr.functions;
 
-import java.util.Properties;
-
-import org.slf4j.LoggerFactory;
 import org.testng.Assert;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import com.google.refine.RefineTest;
 import com.google.refine.expr.EvalError;
+import com.google.refine.grel.GrelTestBase;
 
-public class CoalesceTests extends RefineTest {
+public class CoalesceTests extends GrelTestBase {
 
     private static final Integer[] ZERO_TO_TWO = new Integer[] { 0, 1, 2 };
 
-    @Override
-    @BeforeTest
-    public void init() {
-        logger = LoggerFactory.getLogger(this.getClass());
-    }
-
-    @BeforeMethod
-    public void setUp() {
-        bindings = new Properties();
-    }
-
-    @AfterMethod
-    public void tearDown() {
-        bindings = null;
-    }
-
```

Comment:
I have also used this opportunity to clean up unnecessary set up functions which are already included in the base class.